adapters/epics.py

```diff
@@ -68,9 +69,14 @@ def doProcess(self, dt):
 
 
 class EpicsAdapter(Adapter):
-    def run(self, target, bindings, *args, **kwargs):
+    def run(self, target, bindings, arguments):
+        print arguments
```

Comment:
This should probably be removed otherwise we'll discover weird things later on as it happened last time :)